client/src/elm/MassiveDecks/Pages/Lobby/Configure.elm

```diff
@@ -560,47 +585,6 @@ startGameProblems shared wrap wrapLobby users model remote =
                 |> Maybe.justIf (humanPlayerCount < 1)
             ]
 
-        disableRandoConfig =
```

Comment:
Not sure if I should bring this back but check for "blank white cards only" instead to fix the above comment? And then I guess I'd need to hide the above error message I added in that case too...